lib/components/user/monitored-trip/trip-notifications-pane.tsx

```diff
@@ -233,53 +271,40 @@ class TripNotificationsPane extends Component {
                 <DurationOptions defaultValue={5} minuteOptions={[5, 10, 15]} />
               </FormControl>
             </li>
-          </SettingsListWithAlign>
+          </SettingsList>
 
-          <h4>
-            <SettingsToggle
-              aria-expanded={showAdvancedSettings}
-              aria-label={intl.formatMessage({
-                id: 'components.TripNotificationsPane.toggleAdvancedSettings'
-              })}
-              onClick={this._handleToggleAdvancedSettings}
-              type="button"
-            >
-              <StyledIconWrapper>
-                {showAdvancedSettings ? <CaretDown /> : <CaretRight />}
-              </StyledIconWrapper>
+          <details>
```

Comment:
I believe it's this change that's making it difficult to see that advanced settings can be expanded! 

<img width="809" alt="Screenshot 2023-02-17 at 10 16 57 AM" src="https://user-images.githubusercontent.com/86619099/219693373-60e96133-fbb3-476f-9b8d-bc2bab35092b.png">

Reply:
Is this on a particular browser?

Reply:
Oh yes my bad this is Firefox

Reply:
Thanks for checking on Firefox! Fixed in https://github.com/opentripplanner/otp-react-redux/pull/795/commits/9e81ac57b0848a1f1772700f43a6318d1288488d and that should still work for Chromium and Safari.